src/App.jsx

```diff
@@ -1,10 +1,100 @@
 import { useState } from "react";
```

Comment:
Since it is recommended not to have a component that has more than 80 lines of codes then you can work on a way to break down this large component.